presto-product-tests/src/main/java/io/prestosql/tests/iceberg/TestIcebergCreateTable.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.iceberg;
+
+import io.prestosql.tempto.ProductTest;
+import org.testng.annotations.Test;
+
+import java.sql.SQLException;
+
+import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tests.TestGroups.ICEBERG;
+import static io.prestosql.tests.TestGroups.STORAGE_FORMATS;
+import static io.prestosql.tests.utils.QueryExecutors.onPresto;
+
+public class TestIcebergCreateTable
```

Comment:
It would be nice to have some generic connector tests, that we could run for hive and iceberg (and potentially for other connectors too in future).

Reply:
The scope of the PR is to add iceberg connector to the product tests, so I limited scope of iceberg tests to a bare minimum (create/delete table and run simple query). I propose deferring generalization between hive and iceberg tests and extending iceberg tests scope to be on par with hive tests to a follow up PRs.